governance/program/src/state/realm.rs

```diff
@@ -194,6 +208,20 @@ pub fn get_governing_token_holding_address(
     .0
 }
 
+/// Asserts given realm config args are correct
+pub fn assert_valid_realm_config_args(config_args: &RealmConfigArgs) -> Result<(), ProgramError> {
+    match config_args.community_mint_max_vote_weight_source {
+        MintMaxVoteWeightSource::SupplyFraction(fraction) => {
+            if !(1..=MintMaxVoteWeightSource::SUPPLY_FRACTION_BASE).contains(&fraction) {
```

Comment:
nit: does this perform well for `SUPPLY_FRACTION_BASE` in terms of compute units? I would have just done `1 <= fraction && fraction <= SUPPLY_FRACTION_BASE` to avoid generating a range or iterating through so many values.  I guess the compiler knows how to deal with this

Reply:
Yes, it creates a range with start and end and checks only them. 
This change was suggested by clippy [manual_range_contains](https://rust-lang.github.io/rust-clippy/master/#manual_range_contains)